test/integration/advanced_search_test.rb

```diff
@@ -24,6 +24,8 @@ class AdvancedSearchTest < SystemTest
   end
 
   test "enter inside any field will submit form" do
+    import_and_refresh
```

Comment:
I have added those `import_and_refresh` calls in this test file since those tests are using Searchkick and refreshing the index at the beginning (even there is no gem present) makes it predictable. Searchkick otherwise emits warning if data are out of sync.